### ISSUE TITLE
The bazel build tool is not working on ubuntu only on mac but the data is on ubuntu.

### ISSUE THREAD

**Issue by YanLiang1102** [oudalab/co-incidents#24]

**Comment by YanLiang1102** [comment]
@cegme

**Comment by cegme** [comment]
Can you show the error and what you tried.

**Comment by YanLiang1102** [comment]
let me reintsall bazel after it can not build i did bazel clean then it crashed , it used to be one of the cfile can not get compiled here

**Comment by cegme** [comment]
Try including `<cstring>`

**Comment by YanLiang1102** [comment]
good point after include that is working!

**Comment by YanLiang1102** [comment]
but why it works on Mac without including the cstring?

**Comment by YanLiang1102** [comment]
when I tried to run it again, it failed as the error i saw before, but the target co_yan is in the BUILD_CONFIG, not sure what is complain about , not find a good resource online to solve this
yan@manchester:~/hanover_backup/coincidence/cppcode/bazel_workspace/co-incidents/code$ bazel build co_yan --verbose_failures --sandbox_debug
INFO: Invocation ID: dfef7436-37af-47a8-af2f-c21a1e1152fa
ERROR: Analysis of target '//code:co_yan' failed; build aborted: Analysis failed
INFO: Elapsed time: 0.065s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)

**Comment by cegme** [comment]
Compile it from the workspace directory.

**Comment by YanLiang1102** [comment]
yes I am in the workspace it worked on mac, i think ubuntu has its issue for bazel.

**Comment by YanLiang1102** [comment]
I am not so worried about the build system for now , our clustering result is not promising since from what it clustered, it is almost only on the same day

**Comment by cegme** [comment]
Please paste your new error. You should build from the same directory as the WORKSPACE file.

**Comment by YanLiang1102** [comment]
I did

**Comment by cegme** [comment]
Is the WORKSPACE file in the co-incidents/code folder? It shouldn’t be.

**Comment by YanLiang1102** [comment]
I should build from where the BUILD config file is not where the WORKSPACE is,

**Comment by YanLiang1102** [comment]
see it is working on mac and u should build from where the BUILD config is 
![image](https://user-images.githubusercontent.com/10291607/50554910-4d0fca00-0c78-11e9-9006-e2ba56596b3b.png)

**Comment by cegme** [comment]
I got it to work on manchester. It may have been your install. For another project I put the build call in the workspace, your are right.

**Comment by YanLiang1102** [comment]
yep it is working now great! Thanks Dr.Grant, yeah might be the way I install it.
